source/androidx.core/core/Transforms/EnumMethods.xml

```diff
@@ -18,12 +18,63 @@
         <method jni-name="startNestedScroll" parameter="p0" clr-enum-type="Android.Views.ScrollAxis" />
     </mapping>
     <mapping jni-interface="androidx/core/view/NestedScrollingParent">
+        <!--
         <method jni-name="onNestedScrollAccepted" parameter="p2" clr-enum-type="Android.Views.ScrollAxis" />
-        <method jni-name="onStartNestedScroll" parameter="p2" clr-enum-type="Android.Views.ScrollAxis" />
+        -->
+        <method jni-name="onStartNestedScroll" parameter="axis" clr-enum-type="Android.Views.ScrollAxis" />
```

Comment:
I don't know if it's worth applying this enum, it's going to be a breakage for anyone using these methods, which appear to be used in several packages, like `androidx.constraintlayout:constraintlayout` and `androidx.coordinatorlayout:coordinatorlayout`:

```
#### Type Changed: AndroidX.CoordinatorLayout.Widget.CoordinatorLayout

Removed methods:

public virtual void OnNestedScrollAccepted (Android.Views.View child, Android.Views.View target, int nestedScrollAxes, int type);
public virtual bool OnStartNestedScroll (Android.Views.View child, Android.Views.View target, int axes, int type);


Added methods:

public virtual void OnNestedScrollAccepted (Android.Views.View child, Android.Views.View target, Android.Views.ScrollAxis nestedScrollAxes, int type);
public virtual bool OnStartNestedScroll (Android.Views.View child, Android.Views.View target, Android.Views.ScrollAxis axes, int type);
```

Reply:
let me see if I can remove that.

changes in `androidx.core.core` forced me to add those enum mappings and it simply started to spread....

Reply:
Adding wrappers in `Additions`

I will ask you to re-review this PR again, later.